nss-cohort-40/banagzon-workforce-mgt-lumberjacks
# Bc emp details

# Description
Employee list populates current computer
Employee Details 
Choose to assign a computer

Please delete options that are not relevant.
# Testing Instructions
See if assigned computer displays in list view
If Assigned a computer it should be removed from dropdown
# Checklist:
